### ISSUE TITLE
Edit Image- Will not crop

### ISSUE THREAD

**Issue by JKJensen** [BYU-ARCLITE/Ayamel-Examples#341]
The Edit Image feature currently will not crop, however the rotation changes do work.

**Comment by dro248** [comment]
Hey @gliese1337 and @JKJensen , is this still functionality that we have/want? If so, how does one go about cropping?

**Comment by gliese1337** [comment]
@dro248 Yes, it is, but you need an Image content object to do it with. So, first find or create a Image content object, and then there should be some editing options for it. But if there aren't, then that means code got degraded due to being ignored while other stuff changed, and we will need to add that back at some point.

**Comment by JKJensen** [comment]
@gliese1337  @dro248 It may have gotten degraded. Every time I upload an image content, I get a blank content object area, no matter the browser.
Sample image on [Ayamel](https://ayamel.byu.edu/content/3913) and [AyamelBeta](https://ayamelbeta.byu.edu/content/692).

<img width="677" alt="screenshot 2016-05-03 17 55 24" src="https://cloud.githubusercontent.com/assets/5481718/15001840/3d3dc74e-1158-11e6-81d7-3d55df6a70a5.png">

Oddly enough, the thumbnail of the image works fine.
<img width="604" alt="screenshot 2016-05-03 17 55 51" src="https://cloud.githubusercontent.com/assets/5481718/15001847/4fb22e38-1158-11e6-849a-30e76ba26402.png">

**Comment by stgreenman** [comment]
Hey Jeff and Logan! How are you guys?

When you have a quick second, will you remove me from Ayamel's calendar and
email list?

Thanks!
